src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -25,81 +31,228 @@
 
 USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:86.0) Gecko/20100101 Firefox/86.0'
 
+GENOME_STATUS_FILENAME = "genome_status.txt"
 
-class GenomeStatus:
 
-    def __init__(self, original_id, resolved_id, attempted=False, filename=""):
+class GenomeStatus:
+    """Defines the status relative to a certain genome ID.
+
+    Attributes:
+        original_id(str): original ID for the genome. 
+        resolved_refseq_id(str): ID after having applied _resolve_refseq_id. 
+        attempted(bool): if the attempt of resolving the genome ID has already been done. Defaults to False. 
+        filename(str|PathLike): file in which the data for the genome, if present, are saved. Defaults to "". 
+        """
+
+    def __init__(self,
+                 original_id: str,
+                 resolved_refseq_id: str,
+                 attempted: str | bool = False,
+                 filename: str = ""):
         self.original_id = ';'.join(original_id.split(','))
-        self.resolved_id = None if resolved_id == 'None' else resolved_id
-        self.attempted = True if attempted == 'True' else False
+        self.resolved_refseq_id = "" if resolved_refseq_id == 'None' else resolved_refseq_id
+        if attempted == 'True':
+            self.attempted = True
+        else:
+            self.attempted = False
         self.filename = filename
 
-    @classmethod
-    def from_csv(cls, original_id, resolved_id, attempted, filename):
-        return cls(original_id, resolved_id, attempted, filename)
-
-    def to_csv(self):
-        return ','.join([
+    def add_line(self, file: Path):
+        """Adds a line to file, containing info for the current genome ID. 
+        """
+        line = ','.join([
             str(self.original_id),
-            str(self.resolved_id),
-            str(self.attempted), self.filename
+            str(self.resolved_refseq_id),
+            str(self.attempted),
+            str(self.filename)
         ])
 
+        with open(file, 'a+') as f:
+            f.write(line + '\n')
```

Comment:
```suggestion
    def to_csv(self, file: str | PathLike, mode: 'a') -> None:
        """Write object to a csv file.
        """
        line = ','.join([
            self.original_id,
            self.resolved_refseq_id,
            str(self.attempted),
            self.filename
        ])

        with open(file, mode) as f:
            f.write(line + '\n')
```
A new user will interpret `add_line` as adding a line to the GenomeStatus object.
Please rename the places where it is used.

Reply:
`add_line` writes to a txt file, in particular, it adds a line to a txt file. I think that `to_csv` is not a good name as well. Maybe `to_txt`?

Reply:
the nature of that txt file is csv, so I'd suggest changing the file name to `genome_status.csv`